server/config/runtime.exs

```diff
@@ -330,14 +330,13 @@ config :tuist, Oban,
     {Oban.Plugins.Lifeline, rescue_after: to_timeout(minute: 30)},
     {Oban.Plugins.Cron,
      crontab:
-       if(Tuist.Environment.tuist_hosted?() and env == :prod,
+       if(Tuist.Environment.tuist_hosted?(),
          do: [
            {"0 10 * * 1-5", Tuist.Ops.DailySlackReportWorker},
            {"0 * * * 1-5", Tuist.Ops.HourlySlackReportWorker},
```

Comment:
will these be sending messages to the `#notifications` Slack channel? It might be better to have a dedicated slack channel for these, so we don't pollute our current `#notifications` channel that I actually somewhat actively monitor to see what's happening.

Reply:
I created another channel, `#notifications-non-prod` and pointed the notifications to that channel.